contentcuration/contentcuration/test_settings.py

```diff
@@ -4,6 +4,11 @@
 
 DEBUG = True
 
+if RUNNING_TESTS:
```

Comment:
Nice! What's the ideal way to set this variable?

Reply:
As in better than the way we set it currently? Not sure if there is some attribute, etc. that is always set when tests are running that we could check for.

Reply:
Ah! Looks like `RUNNING_TESTS` is set in `settings.py`, when we call `pytest` or `test`.